content/docs/install/linux.md

```diff
@@ -51,13 +51,17 @@ Depending on the type of the [remote storage](/doc/command-reference/remote) you
 plan to use, you might need to install optional dependencies: `dvc-s3`,
 `dvc-azure`, `dvc-gdrive`, `dvc-gs`, `dvc-oss`, `dvc-ssh`.
```

Comment:
Hi @efiop. So on this (`conda-forge` paks) what's the default? I'm guessing `dvc` includes everything in this case? Unlike `pip`.

Reply:
It used to before 2.0, but now it is same as `pip` - `dvc` package doesn't include anything except for built-ins like http (same as pip package)